workflow-runtime/src/commonMain/kotlin/com/squareup/workflow1/RenderWorkflow.kt

```diff
@@ -133,7 +138,7 @@ public fun <PropsT, OutputT, RenderingT> renderWorkflowIn(
       // It might look weird to start by consuming the output before getting the rendering below,
       // but remember the first render pass already occurred above, before this coroutine was even
       // launched.
-      val output = runner.nextOutput()
+      val output: WorkflowOutput<OutputT>? = runner.processActions()
```

Comment:
internal to the `processActions` implementation, when we would otherwise be getting `null` we don't know the `OutputT` type of which we have a null instance. I don't think it will matter.